python/example_code/glue/README.md

```diff
@@ -0,0 +1,116 @@
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single action
+
+* [Create a crawler](glue_wrapper.py)
+* [Create a job definition](glue_wrapper.py)
+* [Delete a crawler](glue_wrapper.py)
+* [Delete a database from the &GLUDC;](glue_wrapper.py)
+* [Delete a job definition](glue_wrapper.py)
+* [Delete a table from a database](glue_wrapper.py)
+* [Get a crawler](glue_wrapper.py)
+* [Get a database from the &GLUDC;](glue_wrapper.py)
+* [Get a job run](glue_wrapper.py)
+* [Get runs of a job](glue_wrapper.py)
+* [Get tables from a database](glue_wrapper.py)
+* [List job definitions](glue_wrapper.py)
+* [Start a crawler](glue_wrapper.py)
+* [Start a job run](glue_wrapper.py)
+
+### Scenarios
+
+* [Get started running crawlers and jobs](scenario_getting_started_crawlers_and_jobs.py)
+
+## Running the examples
+
+### Get started running crawlers and jobs scenario
+
+This interactive scenario runs at a command prompt and shows you how to use 
+AWS Glue to:
+
+1. Create and run a crawler that crawls a public Amazon Simple Storage
+   Service (Amazon S3) bucket and generates a metadata database that describes the
+   CSV-formatted data it finds.
+2. List information about databases and tables in your AWS Glue Data Catalog.
+3. Create and run a job that extracts CSV data from the source Amazon S3 bucket,
+   transforms it by removing and renaming fields, and loads JSON-formatted output into
+   another Amazon S3 bucket.
+4. List information about job runs and view some of the transformed data.
+5. Delete all resources created by the demo.
+
+This scenario requires the following scaffold resources that are defined in the
+accompanying AWS CloudFormation script `setup_scenario_getting_started.yaml`.
+
+* An Amazon S3 bucket that can contain the Python ETL job script and can receive 
+output data.
+* An AWS Identity and Access Management (IAM) role that can be assumed by AWS Glue. 
+The role must grant read-write access to the S3 bucket and standard rights needed by 
+AWS Glue.
+
+You can deploy the scaffold resources at a command prompt.
+
+```
+python scaffold.py deploy
+```
+
+This outputs a role and bucket name similar to the following.
+
+```
+Outputs:
+        RoleName: AWSGlueServiceRole-DocExample
+        BucketName: doc-example-glue-scenario-docexampleglue6e2f12e5-3zjkuexample
+```
+
+Start the scenario at a command prompt, passing it the role and bucket name.
```

Comment:
Start the scenario at a command prompt, passing the role and bucket name to it.

Reply:
fixed